unblob/handlers/compression/gzip.py

```diff
@@ -167,11 +172,7 @@ def is_valid_gzip(self, path: Path) -> bool:
 
     def calculate_multifile(self, file: Path) -> Optional[MultiFile]:
         paths = sorted(
-            [
-                p
-                for p in file.parent.glob(f"{file.stem}.*")
-                if p.resolve().exists() and p.stat().st_size > 0
-            ]
```

Comment:
Please note, that if this were the only change (and a test with an empty file), this would have been a good fix.

As it is also possible to get it right at a lower level function, this filtering is no longer necessary.